commands/start.go

```diff
@@ -96,6 +98,13 @@ func (s *KoolStart) Execute(args []string) (err error) {
 		}
 	}
 
+	if len(s.Flags.Profile) > 0 {
+		if aware, ok := s.start.(compose.LocalDockerComposeAware); ok {
```

Comment:
Why have you felt the need to introduce this new interface? why not simply call `s.start.AppendArgs("--profile", s.Flags.Profile)`?

Reply:
Because the profile command is an argument to the docker-compose command, if I do whou you sugested, the result will be `docker-compose up --profile profile-1 -d` and this is an invalid command, the valid command is `docker-compose --profile profile-1 up -d`, so I created this new interface, which gives me access to the `localDockerCompose` property which is a command for `docker-compose`, that's where I append the args, because it would be a layer before star and I understand that it would be responsible for play the start before the up, even this attribute receives cmd up.

Reply:
In issue #391 I send an image of the result of this code. But I couldn't create a test with this structure.

My first attempt was to add to attach, as you suggested, but i received invalid arguments from `docker-compose up`